internal/v1/api.yaml

```diff
@@ -1301,6 +1322,27 @@ components:
           type: string
         signature:
           type: string
+    RecommendPackageRequest:
+      required:
+        - packages
+        - recommendedPackages
+      type: object
+      properties:
+        packages:
+          type: array
+          items:
+            type: string
+        recommendedPackages:
+          type: integer
+          format: int32
+          default: 3
+    RecommendationsResponse:
+      type: object
+      properties:
+        packages:
```

Comment:
This field should be required in the response so the caller does not need to deal with pointers.